webtau-cli-groovy/pom.xml

```diff
@@ -25,8 +25,14 @@
 
         <dependency>
             <groupId>org.codehaus.groovy</groupId>
-            <artifactId>groovy-all</artifactId>
-            <type>pom</type>
+            <artifactId>groovy-groovydoc</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>junit</groupId>
```

Comment:
Junit 4?

Reply:
yeah, most of the tests in webtau to test webtau are in junit4.